converter/commands/convert.go

```diff
@@ -267,7 +267,8 @@ func interruptContext(log logging.Logger) (context.Context, func()) {
 	go func() {
 		select {
 		case <-sigChan:
-			log.Info("CTRL-C Received", nil)
+			logTime := time.Now().Format("2006-01-02T15:04:05Z07:00")
+			log.Info(logTime+" CTRL-C Received", nil)
```

Comment:
I believe this will produce a message with two timestamps. The timestamps that you've added in the logrus logger should be sufficient.